### ISSUE TITLE
MarkdownStyle is being overridden in constructor

### ISSUE THREAD

**Issue by rzinnatullin** [whistyun/MdXaml#34]
There is `MarkdownStyle` property in `MarkdownScrollViewer` control.
It can be set directly, or via a style.
However, setting it via a style does not work, because its value is being overridden by `Standard` Markdown style:
```C#
        public MarkdownScrollViewer()
        {
            Engine = new Markdown();
            SetCurrentValue(MarkdownStyleNameProperty, nameof(MdStyle.Standard));
        }
```
The overriding is happening in `MarkdownStyleNameProperty` change handler (`UpdateStyleName` method):
```C#
owner.MarkdownStyle = (Style)prop.GetValue(null);
```

Please fix the issue.

**Comment by rzinnatullin** [comment]
To workaround the issue I've created a descendant class and added the `MarkdownStyleFixed` property which I set in a style:

```C#
    public class MarkdownViewer : MarkdownScrollViewer
    {
        public static readonly DependencyProperty MarkdownStyleFixedProperty =
            DependencyProperty.Register(
                nameof(MarkdownStyleFixed),
                typeof(Style),
                typeof(MarkdownViewer),
                new PropertyMetadata(null, UpdateMarkdownStyleFixed));

        public Style? MarkdownStyleFixed
        {
            get => (Style)GetValue(MarkdownStyleFixedProperty);
            set => SetValue(MarkdownStyleFixedProperty, value);
        }

        private static void UpdateMarkdownStyleFixed(DependencyObject d, DependencyPropertyChangedEventArgs e)
        {
            var control = (MarkdownScrollViewer)d;
            control.MarkdownStyle = (Style)e.NewValue;
        }

        public override void EndInit()
        {
            base.EndInit();
            if (MarkdownStyleFixed != null && MarkdownStyleFixed != MarkdownStyle)
            {
                MarkdownStyle = MarkdownStyleFixed;
            }
        }
    }
```

Hope this helps to someone until the original issue is fixed.

**Comment by whistyun** [comment]
I released v1.15.
Setting a value to MarkdownStyleName in constructor is removed.
